middleware/auth.js

```diff
@@ -0,0 +1,22 @@
+import jwt from 'jsonwebtoken';
+
+export default (req, res, next) => {
+  const { authorization } = req.headers;
+  if (!authorization) {
+    return res.status(401).json({
+      success: false,
+      message: 'You are required to login to access this endpoint',
+    });
+  }
+  const token = authorization.split(' ')[1];
+  jwt.verify(token, process.env.JWT_SECRET, (err, response) => {
+    if (err) {
+      return res.status(401).json({
+        success: false,
+        message: 'Failed to authenticate',
```

Comment:
I think we can also change this error message to something the user can understand also. For example, we can say `Your session has expired, please login again to continue`

Reply:
I get your point. That is a smart one.